tests/benchmark/cmd/fixture.go

```diff
@@ -0,0 +1,167 @@
+)
+
+const (
+	cfgNodeBinary    = "node.binary"
+	cfgRuntimeLoader = "runtime.loader"
+	cfgRuntimeBinary = "runtime.binary"
+	cfgRuntimeID     = "runtime.id"
+)
+
+func fixture() *oasis.NetworkFixture {
+	var runtimeID common.Namespace
+	rtID := viper.GetString(cfgRuntimeID)
+	if err := runtimeID.UnmarshalHex(rtID); err != nil {
+		cmdCommon.EarlyLogAndExit(fmt.Errorf("invalid runtime ID: %s: %w", rtID, err))
+
+	}
+	computeExtraArgs := []oasis.Argument{
+		{
+			Name:   p2p.CfgP2PPeerOutboundQueueSize,
+			Values: []string{"100_000"},
+		},
+		{
+			Name:   p2p.CfgP2PValidateQueueSize,
+			Values: []string{"100_000"},
+		},
+		{
+			Name:   p2p.CfgP2PValidateConcurrency,
+			Values: []string{"100_000"},
+		},
+		{
+			Name:   p2p.CfgP2PValidateThrottle,
+			Values: []string{"100_000"},
+		},
+	}
+	// TODO: runtime.
+	clientExtraArgs := []oasis.Argument{
+		// We manually set the client runtime, as we only set the cupproted
+		// runtime flag, to skip CheckTx on the client.
+		{
+			Name:   runtimeRegistry.CfgSupported,
+			Values: []string{rtID},
+		},
+	}
+	clientExtraArgs = append(clientExtraArgs, computeExtraArgs...)
+
+	fixture := &oasis.NetworkFixture{
+		Network: oasis.NetworkCfg{
+			NodeBinary:             viper.GetString(cfgNodeBinary),
+			RuntimeSGXLoaderBinary: viper.GetString(cfgRuntimeLoader),
+			Consensus: consensusGenesis.Genesis{
+				Parameters: consensusGenesis.Parameters{
+					TimeoutCommit: 1 * time.Second,
+				},
+			},
+			Beacon: beacon.ConsensusParameters{
+				Backend: beacon.BackendInsecure,
+			},
+			HaltEpoch:    10000,
+			FundEntities: true,
+		},
+		Entities: []oasis.EntityCfg{
+			{IsDebugTestEntity: true},
+			{},
+		},
+		Validators: []oasis.ValidatorFixture{
+			{Entity: 1},
+		},
+		Clients: []oasis.ClientFixture{
+			{NodeFixture: oasis.NodeFixture{ExtraArgs: clientExtraArgs}},
+		},
+		StorageWorkers: []oasis.StorageWorkerFixture{
+			{Entity: 1, Runtimes: []int{0}, Backend: badger.BackendName},
+		},
+		ComputeWorkers: []oasis.ComputeWorkerFixture{
+			{Entity: 1, Runtimes: []int{0}, NodeFixture: oasis.NodeFixture{ExtraArgs: computeExtraArgs}},
+			{Entity: 1, Runtimes: []int{0}, NodeFixture: oasis.NodeFixture{ExtraArgs: computeExtraArgs}},
+			{Entity: 1, Runtimes: []int{0}, NodeFixture: oasis.NodeFixture{ExtraArgs: computeExtraArgs}},
+		},
+		Seeds: []oasis.SeedFixture{{}},
+		Runtimes: []oasis.RuntimeFixture{
+			{
+				ID:         runtimeID,
+				Kind:       registry.KindCompute,
+				Entity:     0,
+				Keymanager: -1,
+				Binaries: map[node.TEEHardware][]string{
+					node.TEEHardwareInvalid: {viper.GetString(cfgRuntimeBinary)},
+				},
+				Executor: registry.ExecutorParameters{
+					GroupSize:       2,
+					GroupBackupSize: 1,
+					RoundTimeout:    5,
+					MaxMessages:     128,
+				},
+				TxnScheduler: registry.TxnSchedulerParameters{
+					Algorithm:         registry.TxnSchedulerSimple,
+					MaxBatchSize:      10_000,
+					MaxBatchSizeBytes: 10 * 16 * 1024 * 1024, // 160 MiB
```

Comment:
Whoa, these are some large batches!

Reply:
hm yeah i think i copied this from some other benchmarks, the batches aren't close this big though, the batch size is reached before.